mainchain-follower/db-sync-follower/src/candidates/mod.rs

```diff
@@ -133,19 +133,36 @@ impl CandidateDataSource for CandidatesDataSourceImpl {
 		Ok(AriadneParameters { d_parameter: d_param, permissioned_candidates: candidates })
 	}
 
+	// Get all registered candidate registrations for the given epoch.
+	// Note, this method requests only ADA candidates from Cardano and
 	async fn get_candidates(
 			&self,
 			epoch: McEpochNumber,
 			committee_candidate_address: MainchainAddress
 		)-> Result<Vec<CandidateRegistrations>> {
 		let epoch = EpochNumber::from(self.get_epoch_of_data_storage(epoch)?);
-		let candidates = self.get_registered_candidates(epoch, committee_candidate_address).await?;
+
+		// get registered candidates for ADA
+		let ada_candidates = self
+			.get_registered_candidates(epoch, committee_candidate_address)
+			.await?;
+
+		// TODO ETH: get registered candidates for ETH
```

Comment:
D-param lives on Cardano, Ada candidates also live on Cardano. Cardano = Mainchain. We can rename Mainchain to Cardano (I still doubt we should), but we cannot have two Mainchains for sure. My point is that we perhaps should not get ETH candidates from Cardano (Mainnet)

Reply:
Yeah, somewhat misleading description, fixed. It make sense to continue calling Cardano as Mainchain since its role is special for governance. On the other hand, from virtual stake calculation point of view Cardano's role is symmetric to Ethereum.

Reply:
:+1: My understanding is: for Minotaur there will be many sources of stake that are equal. But Cardano is the source of DParam. Is it correct?

Reply:
Yes, besides ETH, there are talks to add PC's native token. But governance will be done on Cardano, so it is to some degree the "main" chain.

Reply:
we will need to consider cases where other chains are the "main chain" as well as where the main chain is the partner chain itself.  not sure it needs to be concerned with right now.